traffic_ops/traffic_ops_golang/profileparameter/profile_parameters.go

```diff
@@ -44,8 +44,9 @@ type TOProfileParameter struct {
 	tc.ProfileParameterNullable
 }
 
-func (v *TOProfileParameter) NewReadObj() interface{} { return &tc.ProfileParametersNullable{} }
-func (v *TOProfileParameter) SelectQuery() string     { return selectQuery() }
+func (v *TOProfileParameter) AllowMultipleCreates() bool { return true }
```

Comment:
`TOProfileParameter.AllowMultipleCreates()` has the same name as the `v1.AllowMultipleCreates` interface. Should one of these names change?

Also, please comment this function.

Reply:
done